osu.Game.Rulesets.Tau/UI/TauCachedProperties.cs

```diff
@@ -23,10 +21,6 @@ public void SetRange(float cs)
             AngleRange.Value = IBeatmapDifficultyInfo.DifficultyRange(cs, 75, 25, 10);
         }
 
-        public void Dispose()
-        {
-            SliderTexture?.Dispose();
-            SliderTexture = null;
-        }
+        public void Dispose() {}
```

Comment:
Since there is no longer anything to `Dispose`, this method can be removed, and this class doesn't need to be an `IDisposable`